osu.Game/Screens/Edit/Compose/SelectionHelper.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using System.Text;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Platform;
+using osu.Game.Extensions;
+using osu.Game.Rulesets.Edit;
+
+namespace osu.Game.Screens.Edit.Compose
+{
+    public class SelectionHelper : Component
```

Comment:
Why is this its own component? I don't see any reason for it to be. Also we don't use the `Helper` suffix in drawable hierarchy components, ever.

Reply:
I was pushing for it to be a component to reduce overall noise in already large editor classes. Do you think it would be better off to have this somewhere else specifically?

Reply:
Considering it's a single method, I'd probably just put it in an existing handler class. Right now this is adding more noise.